### DISCUSSION TITLE
want my extensions to start when I click on run Workflow

### DISCUSSION THREAD

**Discussion by Atharv115** [AutomaApp/automa#500] [Ideas]
@Kholid060 It just stiked to me.
Whenever I open my browser it takes longer time than usual (in the sense, when there were no extensions) it loads my browser faster when there are no extensions, but as we all know extensions make our life easier while browsing.

So what I wanted was to create such a workflow, where when I click on run workflow it should enable all my required extensions (the simple reason being the impact the extensions had on the startup time of the browser).btw when I clicked on record workflow it just recorded that I visited the extensions page but it didn't register the toggle's that I made to turn on my required extension (obviously for the purpose of recording the workflow I had disabled the required ones and while recording the workflow put it back on)

to be frank to @Kholid060  you, when I need to create a workflow, I just record my workflow and save it, coz I don't have that deep knowledge, like if u are willing u could walk me through the process to make the above workflow, or do the above workflow and share it in your marketplace. 

please do respond to this Idea that I wanted to share
![bitmoji](https://sdk.bitmoji.com/render/panel/12f3a5e1-c85f-49cd-94f4-6dce9cb18ef1-d24f89b5-ba29-4739-bfb0-352799c3c9cd-v1.png?transparent=1&palette=1&width=246)
Lots of Love ❤️❤️, Really liked ur work
continue making such good things!!
![bitmoji](https://sdk.bitmoji.com/render/panel/0f789e4d-9133-4194-85af-a55b4027d8b4-d24f89b5-ba29-4739-bfb0-352799c3c9cd-v1.png?transparent=1&palette=1&width=246)

**Comment by Kholid060** [comment]
Unfortunately, Automa is not able to do the workflow you mentioned above. It's because a chrome extension can't access the tab where the URL starts with `chrome` like `chrome://extensions`.

**Reply by Atharv115** [reply]
k, fine hope u find a soluntion to this thing soon

**Reply by Atharv115** [reply]
solution*

**Reply by edwyn-amador** [reply]
> k, fine hope u find a soluntion to this thing soon

@Atharv115  Like it was said this is not even possible from a chrome extension level, what you could do is to use a desktop script for this, you could try with Autohotkey as for example

**Reply by Atharv115** [reply]
K fine @edwyn-amador, What is it?

**Reply by edwyn-amador** [reply]
@Atharv115  It is an automation scripting language for Windows (from a desktop environment level instead of the browser level which is limited), in case you are not using Windows you could search for an alternative for a different OS

**Reply by Atharv115** [reply]
ok
